client/src/models/history/As1_Assembly.ts

```diff
@@ -83,6 +83,9 @@ export const create: Create = async (apiType, params?) => {
     0,
     0,
     0,
+    0,
```

Comment:
Due to this [commit](https://github.com/awv-informatik/buerli/pull/565/commits/a11783b66c9934d8ab3dfea50b0d8a0b5f8e2052) in buerli we can get rid of the most of these changes here, as they are now optional

Reply:
This PR is actually not needed at all with changes from that commit. I should probably close it, or do you think there are changes that are useful?

Reply:
You're right.